hedyorg/hedy
# 🪲 Adds input field for filtering programs

Adds an input field in the filter from, so the user can actually filter the program's in the desired user, and not only their own, also changes the value of the label for no programs to be more generic.

Fixes #4346

**How to test**

* Go to a class and select a student page
* Use the filter
* The programs shown should be the student's
